src/components/screens/management-screen.tsx

```diff
@@ -344,6 +437,66 @@ const ManagementScreen: React.FC = () => {
     );
   };
 
+  /**
+   * Renders billiable hours update dialog
+   */
+  const renderBillableHoursUpdateDialog = () => {
+    if (!selectedPersonWithTotalTime || !selectedPersonWithTotalTime.personTotalTime) {
+      return null;
+    }
+
+    const { person } = selectedPersonWithTotalTime;
+    
+    return (
+      <GenericDialog
+        title={ strings.billableHoursHandling.title }
+        open={ billableHoursUpdate }
+        error={ false }
+        onClose={ () => setBillableHoursUpdate(false) }
+        onCancel={ () => setBillableHoursUpdate(false) }
+        onConfirm={ () => setBillableHoursUpdate(false) }
+      >
+        <Box className={classes.updateBillableHoursContent}>
```

Comment:
Fix space errors from this file. {classes.updateBillableHoursContent} should be { classes.updateBillableHoursContent }

Reply:
Done.